src/gurt/telemetry.c

```diff
@@ -682,6 +707,101 @@ d_tm_build_path(char **path, char *item, va_list args)
 	return rc;
 }
 
+/**
+ * Compute gauge statistics, min, max, mean, and standard deviation
+ * Uses B. P. Welford's algorithm for computing a running variance.
+ * This implementation computes an integer min and max with floating point
+ * mean and standard deviation.
+ *
+ * \param[in]	node		Pointer to a gauge node
+ */
+void
+d_tm_compute_gauge_stats(struct d_tm_node_t *node)
+{
+	uint64_t	value = 0;
+	double		sum_of_squares = 0.0;
+	double		variance = 0.0;
+	double		mean = 0.0;
```

Comment:
Maybe define a var on the stack pointing at dtn_metric to make the code more readable?

Reply:
I made a stack var for dtm_stats instead.